src/main/scala/nl/knaw/dans/easy/fedoratobag/Spatial.scala

```diff
@@ -117,11 +127,21 @@ case class SpatialBox(scheme: Option[String],
   lazy val dcxGml: Option[Elem] = value.map(_ =>
     <dcx-gml:spatial>
       <boundedBy xmlns="http://www.opengis.net/gml">
-          <Envelope srsName={ srsName }>
+          <Envelope srsName={ guessSrsName }>
               <lowerCorner>{ lower }</lowerCorner>
               <upperCorner>{ upper }</upperCorner>
           </Envelope>
       </boundedBy>
     </dcx-gml:spatial>
   )
+
+  private lazy val guessSrsName: String = srsName match {
+    case null => {
+      if(Seq(sNorth, sSouth).map(p => p.toDouble).max > 289000)
+        SpatialNames.RD_SRS_NAME
+      else
+        SpatialNames.DEGREES_SRS_NAME
+    }
```

Comment:
Extract val for north/south (cmd-alt-v), then extract method for the if-then-else (cmd-alt-m). The second action will be applied above too, if not I might be overlooking a difference. Then inline north/south again (cmd-alt-n).

Reply:
done